Mathlib/Algebra/Category/AlgebraCat/Basic.lean

```diff
@@ -46,22 +46,88 @@ instance : CoeSort (AlgebraCat R) (Type v) :=
 
 attribute [coe] AlgebraCat.carrier
 
+/-- The object in the category of R-algebras associated to a type equipped with the appropriate
+typeclasses. -/
+def of (X : Type v) [Ring X] [Algebra R X] : AlgebraCat.{v} R :=
+  ⟨X⟩
+
+@[simp]
+lemma coe_of (X : Type v) [Ring X] [Algebra R X] : (of R X : Type v) = X :=
+  rfl
+
+variable {R} in
+/-- The type of morphisms in `AlgebraCat R`. -/
+@[ext]
+structure Hom (A B : AlgebraCat.{v} R) where
+  /-- The underlying algebra map. -/
+  algHom : A →ₐ[R] B
```

Comment:
`toHom` or `toAlgHom` might be a better name, which matches `QuadraticModuleCat.toIsometry`; though we can of course rename that one instead.

Perhaps a zulip poll for how to name these is sensible?

Reply:
Started a zulip poll

Reply:
The poll has a slight favor for `.hom` so I went with that.